src/__tests__/core/index.test.js

```diff
@@ -37,6 +37,15 @@ describe('setup', () => {
     const model = mock.calls[0][1].toJS();
     expect(model.auth.redirectUrl).toBe('https://test.com/path/');
   });
+  it('default redirectUrl should work when `window.location.origin` is not available', () => {
+    setURL('https://test.com/path/#not-this-part', { noOrigin: true });
+    const options = {};
```

Comment:
any reason why you have the declaration separate from the usage, since you're hardcoding the rest of the parameters too?

Reply:
easy to copy/paste later if needed :grimacing: